docs/components/receive.md

```diff
@@ -15,6 +15,14 @@ Thanos Receive supports multi-tenancy by using labels. See [Multitenancy documen
 For more information please check out [initial design proposal](../proposals/201812_thanos-remote-receive.md).
 For further information on tuning Prometheus Remote Write [see remote write tuning document](https://prometheus.io/docs/practices/remote_write/).
 
+Currently in version [v0.21.2](https://github.com/thanos-io/thanos/blob/main/CHANGELOG.md#v0211---20210604), receiver implements the split behaviour of solo ingestion/routing (see the [proposal](https://github.com/thanos-io/thanos/blob/main/docs/proposals/202012_receive_split.md) for more details). This means, we can enable receiver to ingest data only, or re-route requests only.
+
+This helps receiver relieve of additional responsibility, and can focus on solo behaviour of ingestion/routing. This would enable users to prepare a topology of receivers, containing trees of receiver with depth **N**.
```

Comment:
This paragraph starts with `This`, however, it is a new paragraph so it's far away from whatever `This` _is_ and it's not clear what it is referring to. Is `This` the proposal? The Receiver? The implementation of the proposal?

Reply:
I have replaced `this` with `implementation of receiver split proposal`. Let me know if this works :D